lib/pod-names.js

```diff
@@ -0,0 +1,32 @@
+var Plugin = require('broccoli-plugin');
+var walkSync = require('walk-sync');
+var fs = require('fs');
+var path = require('path');
+var componentNames = require('./component-names.js');
+
+module.exports = PodNames;
+
+PodNames.prototype = Object.create(Plugin.prototype);
+PodNames.prototype.constructor = PodNames;
+function PodNames(inputNode, options) {
+  options = options || {};
+  Plugin.call(this, [inputNode], {
```

Comment:
likely want to revisit this as a persistentOutput filter, so the build output remains stable and doesn't invalidate downstream. This can come as a later addition, but will likely be important to maintain performance.